### ISSUE TITLE
Support TypeScript 4.9 `satisfies` operator

### ISSUE THREAD

**Issue by karlhorky** [daidodo/format-imports#17]
Hi @daidodo 👋 Hope that you are well!

**Is your feature request related to a problem? Please describe.**

It would be great to be able to use [the TypeScript 4.9 `satisfies` operator](https://devblogs.microsoft.com/typescript/announcing-typescript-4-9/#satisfies)!

However, currently, `format-imports` identifies this as an unused import:

```ts
import {AType} from '../a.js'; // 💥 This will be removed as an "unused import"

const a = 'abc' satisfies AType;
```

**Describe the solution you'd like**

The import of the type should not be marked as unused.

**Describe alternatives you've considered**

Ignore all type imports using [`keepUnused`](https://github.com/daidodo/format-imports#keeping-unused-imports)

**Additional context**

I'm using VS Code `1.73.1` which has TS `4.8.4` built in, but I'm using `4.9.3` via my `node_modules`:

<img width="590" alt="Screenshot 2022-11-21 at 17 44 56" src="https://user-images.githubusercontent.com/1935696/203112021-a36324f9-d0ae-42e1-bb69-ddd408d22aba.png">

One thing I could imagine is that the `format-import` VS Code extension needs a newer version of VS Code to get full TS 4.9 support... 🤔 Is this how it works?

**Comment by daidodo** [comment]
Thanks for the feedback!

You are probably right that the fix would be updating the built-in TS.

**Comment by daidodo** [comment]
This should be fixed in 7.5.2.

**Comment by karlhorky** [comment]
Nice, it works for me, thanks! 🙌 Closing.

I guess it was mostly this commit?

- https://github.com/daidodo/format-imports/commit/0065e89d237484432661c13e8b50fa4bff3a6eca
